apps/cow-fi/app/(main)/legal/cowswap-terms/page.tsx

```diff
@@ -711,8 +711,8 @@ export default function Page() {
               Must be a real user who conducted trades on CoW Swap on the Base Network during February 2025 and must:
             </p>
             <ul>
-              <li>be minimum 18 years of age.</li>
-              <li>Not be an employee, affiliate, or immediate family member of the Organizer.</li>
+              <li>be minimum 18 years of age;</li>
```

Comment:
Why replacing the dot with a semicolon?

Reply:
Because this patter is used everywhere at the top of the document: 
![image](https://github.com/user-attachments/assets/bd132ddb-bb3b-424f-a22f-2cf231743def)
![image](https://github.com/user-attachments/assets/7b12e8e4-5c4f-47db-8339-020c2c55c49e)
![image](https://github.com/user-attachments/assets/0a1a167a-aea9-4fea-96f8-8fa43319ea9f)